CHANGES.md

```diff
@@ -1,5 +1,7 @@
 ### 2.1.0 (in developement)
 
+- Fixed a thread-safety issue when iterating over the dict of processes in
+  `ReusablePoolExecutor._resize`.
```

Comment:
Maybe update this to: 
Fixed thread-safety issues for `ReusablePoolExecutor._resize` and `ReusablePoolExecutor.shutdown`